src/frontend/Deprecation_analysis.mli

```diff
@@ -0,0 +1,28 @@
+open Core_kernel
+open Ast
+
+val deprecated_userdefined : string String.Table.t
+val distribution_suffix : string -> bool
+val without_suffix : string sexp_list -> string -> string
+val is_distribution : string -> bool
+val rename_distribution : string -> string
+val rename_function : string -> string
+
+val userdef_distributions :
+     ( (located_meta, unit) expr_with
+     , located_meta
+     , ((located_meta, unit) expr_with, located_meta) lval_with
+     , unit )
+     statement_with
```

Comment:
Does this have a short-hand type in the AST module?

Reply:
:-)

I tried `typed_statement` before, and it caused errors in `repair_syntax`. I now realize that this happens before `let typed_ast = ...`, and that `untyped_statement` works, thanks.